README.md

```diff
@@ -1,8 +1,3 @@
-[![Quality assurance](https://github.com/ExtraHorizon/javascript-sdk/actions/workflows/qualilty-assurance.yml/badge.svg?branch=master)](https://github.com/ExtraHorizon/javascript-sdk/actions/workflows/qualilty-assurance.yml)
-[![Code style](https://github.com/ExtraHorizon/javascript-sdk/actions/workflows/code-style.yml/badge.svg?branch=master)](https://github.com/ExtraHorizon/javascript-sdk/actions/workflows/code-style.yml)
-[![Maintainability](https://api.codeclimate.com/v1/badges/baa71ad27c3ce167cd7d/maintainability)](https://codeclimate.com/github/ExtraHorizon/javascript-sdk/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/baa71ad27c3ce167cd7d/test_coverage)](https://codeclimate.com/github/ExtraHorizon/javascript-sdk/test_coverage)
-
```

Comment:
Just double checking here but are we okay with this being gone? otherwise I will have to add back `code-climate` and split up the actions again.